locales-mn-MN.xml

```diff
@@ -70,37 +70,37 @@
 
     <!-- ORDINALS -->
     <term name="ordinal">th</term>
-    <term name="ordinal-01">st</term>
-    <term name="ordinal-02">nd</term>
-    <term name="ordinal-03">rd</term>
-    <term name="ordinal-11">th</term>
-    <term name="ordinal-12">th</term>
-    <term name="ordinal-13">th</term>
+    <term name="ordinal-01">1-р</term>
+    <term name="ordinal-02">2-р</term>
+    <term name="ordinal-03">3-р</term>
+    <term name="ordinal-11">11-р</term>
+    <term name="ordinal-12">12-р</term>
+    <term name="ordinal-13">13-р</term>
```

Comment:
@adam3smith, my Mongolian is a bit rusty, but this looks a bit suspicious. Should probably just be `<term name="ordinal">-p</term>` as sole ordinal term. Agreed?

Reply:
@adam3smith, bump.

Reply:
oh yes, sorry. 
![image](https://user-images.githubusercontent.com/128986/52918123-92886500-32c1-11e9-8811-9e62e16d08d5.png)

Reply:
I'll prepare a fix.

Reply:
(fixed in #189)